packages/utils/src/RBAC.js

```diff
@@ -0,0 +1,93 @@
+
+export async function getRBAC(applicationName = '') {
+    const insights = window.insights;
+    if (
+        insights &&
+    insights.chrome &&
+    insights.chrome.auth &&
+    insights.chrome.auth.getUser &&
+    insights.chrome.getUserPermissions
+    ) {
+        const user = await insights.chrome.auth.getUser();
+        const permissions = await insights.chrome.getUserPermissions(applicationName);
+        return {
+            isOrgAdmin: user.identity.user.is_org_admin,
+            permissions
+        };
+    }
+
+    return { isOrgAdmin: false, permissions: null };
```

Comment:
This can be even more simplified
```JS
export async function getRBAC(applicationName = '') {
  const user = await insights?.chrome?.auth?.getUser()
  return {
    isOrgAdmin: user?.identity?.user?.is_org_admin || false
    permissions: (await insights?.chrome?.getUserPermissions(applicationName)) || null
  }
}
```